README.md

```diff
@@ -30,10 +31,46 @@ url: '[user].github.io/[repo]'
 ```
 
 There is no need to call this package directly, once `config.yaml` has been updated
-[{sandpaper}] will detect it and copy the styling and templates to your lesson website.
+[{sandpaper}] will detect it and copy the styling and templates to your lesson
+website when building in GitHub pages.
 
-Once `config.yaml`, typical guides from The Carpentries can be followed to deploy
-locally or to GitHub pages.
+Once `config.yaml` has been customised, typical guides from The Carpentries can be
+followed to deploy locally or to GitHub pages.
+
+### Applying Varnish locally
+
+When rendering the site locally the varnish will not, by default, be applied since
+it is not available. A few extra steps to setup up a [{renv}] and install the
+necessary packages are required.
+
```

Comment:
I'm enough of an infrequent R user, that it's worth explaining how to install packages!
*Not sure how to include a full code block in a suggestion, end tag is getting eaten and doing something weird.*

```suggestion

e.g. within `rterm`

```r
install.packages('renv')
```

```